capstone-app/src/app/modules/history/history.module.ts

```diff
@@ -1,10 +1,17 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { WikiMediaComponent } from './wiki-media/wiki-media.component';
 
 @NgModule({
-  declarations: [],
+  declarations: [WikiMediaComponent],
   imports: [
     CommonModule
+  ],
+  exports: [
+    WikiMediaComponent
+  ],
+  entryComponents: [
+    WikiMediaComponent
```

Comment:
Why is it an entry component? Will this be loaded in a modal window or something?

Reply:
That one is on me. I'm used to using entryComponents on all of our components, so I suggested it might be needed here too. Apologies for the confusion.

Reply:
So should I leave it?